docs/changelog.md

```diff
@@ -12,6 +12,17 @@ permalink: /changelog/
 * [Gecko](https://github.com/mozilla-mobile/android-components/blob/master/buildSrc/src/main/java/Gecko.kt)
 * [Configuration](https://github.com/mozilla-mobile/android-components/blob/master/buildSrc/src/main/java/Config.kt)
 
+* **browser-menu2**
+  * Added new component to replace **browser-menu**. menu2 uses immutable data types and better integrates with `BrowserStore`.
+
+* **concept-menu**
+  * Added concept component to contain data classes for menu items and interfaces for menu controllers.
+
+* **browser-toolbar**
+  * Added support for the new `MenuController` interface for menu2.
+    When a menu controller is added to a toolbar, it will be used in place of the `BrowserMenuBuilder`.
+    The builder will supply items to the `MenuController` in `invalidateMenu` if it is kept.
```

Comment:
Unless I'm missing something, this doesn't apply to custom menu items, right? [WebExtensionBrowserMenuItem](https://github.com/mozilla-mobile/android-components/blob/master/components/browser/menu/src/main/java/mozilla/components/browser/menu/item/WebExtensionBrowserMenuItem.kt) for instance.

Reply:
Anything that implements `asCandidate` will work automatically. However, I don't plan to rely on this compat behaviour and I want to just migratate addons all at once.